mods/ra/maps/fort-lonestar/map.yaml

```diff
@@ -654,7 +654,7 @@ Rules:
 		Health:
 			HP: 200
 		AutoTarget:
-			InitialStance: Defend
+			InitialBotStance: Defend
```

Comment:
I guess this has to be `InitialStanceAI` to match the code although `InitialBotStance` has better readability.